assets/src/edit-story/components/canvas/singleSelectionMovable.js

```diff
@@ -196,7 +203,9 @@ function SingleSelectionMovable({ selectedElement, targetEl, pushEvent }) {
   );
 
   const canSnap =
-    !snapDisabled && (!isDragging || (isDragging && !activeDropTargetId));
+    !snapDisabled &&
+    !isEditMode &&
+    (!isDragging || (isDragging && !activeDropTargetId));
```

Comment:
Yeah, this one is definitely getting complex. It would make sense with a cleaner API for this.

Reply:
Should improve in #3016